### PR TITLE
fixed readme

### PR DESCRIPTION
**Pull request by sourTaste000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ Why should you use this over other Kotlin bots:
...
 - Config types support providing a URL instead of a file name, allowing you to load configurations remotely
-- Command registration is automatic, just create an object which extends `Command("commandName")`
+- Command registration is automatic, just create an object which extends `BotCommand(name = "Example", description = "Example Command)`
 - Has a DSL for our own Brigadier-like command system, making creating a command *as easy as just writing a few lambdas*
 - Has (optional) automatic updating
 
 ## Contributing
 
-0. `git clone --recurse-submodules git@github.com:kami-blue/bot-kt.git`
+0. `git clone --recurse-submodules https://github.com/kami-blue/bot-kt.git`
```
Comment (l1ving) [inline README.md:26]: why was this changed?

File: README.md [hunk 2]
```diff
@@ -17,13 +17,13 @@ Why should you use this over other Kotlin bots:
 - Really easy to configure
 - Entirely modular, including config types and commands
 - Config types support providing a URL instead of a file name, allowing you to load configurations remotely
-- Command registration is automatic, just create an object which extends `Command("commandName")`
+- Command registration is automatic, just create an object which extends `BotCommand(name = "Example", description = "Example Command)`
```
Comment (l1ving) [inline README.md:20]: the syntax is incorrect here, you're missing a "

**Review by l1ving — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sourTaste000) [reply README.md:26]: Some people might don't have ssh setup

File: README.md (hunk 2 above)
Comment (sourTaste000) [reply README.md:20]: Whops

File: README.md (hunk 1 above)
Comment (l1ving) [reply README.md:26]: you can't even use GitHub over https to push commits anymore, but oh well. I guess it's fine

**Review by l1ving — APPROVED** [review]
lgtm, ty
